app/models/Subscription.ts

```diff
@@ -0,0 +1,33 @@
+import { observable } from "mobx";
+import BaseModel from "./BaseModel";
+import Field from "./decorators/Field";
+
+class Subscription extends BaseModel {
+  @Field
+  @observable
+  id: string;
+
+  userId: string;
+
+  documentId: string;
+
+  event: string;
+
+  enabled: boolean;
```

Comment:
All of these properties above should have the `@Field` and `@observable` decorators, likely not causing any problems rn but good for consistency – never know how the model will be used in the future.